specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -873,6 +873,16 @@
               "type": "boolean"
             }
           }
+        },
+        "dotNet": {
+          "description": ".NET app configuration",
+          "type": "object",
+          "properties": {
+            "autoConfigureDataProtection": {
+              "description": "Auto configure the ASP.NET Core Data Protection feature",
+              "type": "boolean"
```

Comment:
>boolean [](http://example.com/codeflow?start=23&length=7)

You'be probably considered already, but using bool kind of locks you into the on/off model for this.

Reply:
Yes it's correct! We believe it's OK for this feature as we don't foresee any other need for more granularity.